openedx/core/lib/xblock_utils/__init__.py

```diff
@@ -148,9 +149,41 @@ def wrap_xblock(
         template_context['js_init_parameters'] = ""
 
     if isinstance(block, (XModule, XModuleDescriptor)):
+
+        if context.get('include_dependencies', False):
+            for url in compressed_css_urls('style-vendor'):
```

Comment:
We can actually condense this in a for loop, something like this

```
for style_name in ['style-vendor', 'style-main-v1', ...]:
    for url in compressed_css_urls(style_name):
        frad.add_css_url(url)
```

I am not sure if this is correct or not, but it may be possible. 
If this is possible here, then something similar can be tried for `compressed_js_urls` as well.

Reply:
Yup, this is correct.